.github/workflows/main.yml

```diff
@@ -7,34 +7,34 @@ env:
 
 jobs:
   ci-config:
-      runs-on: ubuntu-latest
```

Comment:
Looks good, but you will have to fix the closing bracket in your Signed-off-by: line, and you should prefix the commit subject with "ci" like so:

> ci: fix inconsistent YAML indentation